Src/Support/Google.Apis.Auth.PlatformServices/Google.Apis.Auth.PlatformServices.csproj

```diff
@@ -1,11 +1,16 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
Hmm... don't remember seeing the ToolsVersion part in my project files. Not a problem.